server/fleetv2_http_api/models/payload.py

```
@@ -2,12 +2,12 @@
 from typing import List, Dict  # noqa: F401
 
-from server.fleetv2_http_api.models.base_model import Model
-from server.fleetv2_http_api.models.payload_data import PayloadData
+from fleetv2_http_api.models.base_model import Model
+from fleetv2_http_api.models.payload_data import PayloadData
 import re
-from server.fleetv2_http_api import util
+from fleetv2_http_api import util
 
-from server.fleetv2_http_api.models.payload_data import PayloadData  # noqa: E501
+from fleetv2_http_api.models.payload_data import PayloadData  # noqa: E501
```

Ruff (0.8.2)

Redefinition of unused `PayloadData` from line 6

Remove definition: `PayloadData`

(F811)